src/sentry/runner/commands/configoptions.py

```diff
@@ -0,0 +1,150 @@
+import click
+
+from sentry.runner.decorators import configuration
+
+
+@click.group()
+def configoptions():
+    "Manages Sentry options."
+
+
+@configoptions.command()
+@click.option("--key", "-k", required=True, help="The key of the option to fetch.")
+@configuration
+def fetch(key):
+    "Fetches the option for the given key."
+    click.echo(get(key))
+
+
+@configoptions.command()
+@configuration
+def list():
+    "Fetches all options."
+    from sentry import options
+
+    for opt in options.all():
+        # click.echo(f"{opt.name} ({opt.type}) = {options.get(opt.name)}")
+        click.echo(f"{opt.name}: {options.get(opt.name)}")
+
+
+@configoptions.command()
+@click.argument("filename", required=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    default=False,
+    required=False,
+    help="Output exactly what changes would be made and in which order.",
+)
+@configuration
+def patch(filename, dryrun):
+    "Updates, gets, and deletes options that are each subsectioned in the given file."
+    import yaml
+
+    if dryrun:
+        click.echo("Dryrun flag on. ")
+    with open(filename) as stream:
+        configmap_data = yaml.safe_load(stream)
+        data = configmap_data.get("data", {}).get("options-patch.yaml", "")
+
+        keysToFetch = data.get("fetch", {})
+        keysToFetch = (line for line in keysToFetch.split("\n") if line)
+
+        keysToUpdate = data.get("update", {})
+        keysToUpdate = (line.split(": ") for line in keysToUpdate.split("\n") if line)
+
+        keysToDelete = data.get("delete", {})
+        keysToDelete = keysToFetch = (line for line in keysToDelete.split("\n") if line)
+
+        for key in keysToFetch:
+            click.echo(get(key, dryrun))
+
+        for key, val in keysToUpdate.items():
+            click.echo(set(key, dryrun))
+
+        for key in keysToDelete:
+            click.echo(delete(key, dryrun))
+
+
+@configoptions.command()
+@click.argument("filename", required=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    default=False,
+    required=False,
+    help="Output exactly what changes would be made and in which order.",
+)
+@configuration
+def strict(filename, dryrun):
```

Comment:
Why do we need both patch and strict? Which one will we use in the self service process ?

Reply:
I think the focus is more on strict right now

Reply:
My initial goal is to use strict for _most_ of the options. You've mentioned a couple of different features that use options so we'll need to ignore those key/value pairs in some structured controllable way.